test/parallel_api/algorithm/alg.sorting/sort_by_key.pass.cpp

```diff
@@ -20,6 +20,56 @@
 
 #include "support/utils.h"
 
+template<typename _Policy, typename _KeyIt, typename _ValIt, typename _Size>
+void
+call_sort_by_key(_Policy&& policy, _KeyIt keys_begin, _ValIt vals_begin,  _Size n)
+{
+    auto counting_begin = oneapi::dpl::counting_iterator<int>{0};
+
+    // 1. Initialization of buffers
+    std::transform(policy, counting_begin, counting_begin + n, keys_begin,
+                   [](int i) { return i % 2 + 1; });
+    // fill vals_buf with the analogue of std::iota using counting_iterator
+    std::copy(policy, counting_begin, counting_begin + n, vals_begin);
+
+    // 2. Sorting
+    // stable sort by keys
+    oneapi::dpl::sort_by_key(policy, keys_begin, keys_begin + n, vals_begin, std::less<void>());
+}
+
+template<typename _Keys, typename _Vals, typename _Size>
+void
+check_sort_by_key_results(const _Keys& keys_buf, const _Vals& vals_buf,  _Size n)
+{
+    //Checking results
+    constexpr int k = (n - 1) / 2 + 1;
```

Comment:
I believe it doesn't compile anymore. Either it's template parameter and then it could be `constexpr` or it's just `const`

Reply:
yes, I returned it to const